packages/studiocms/src/components/user-quick-tools.ts

```diff
@@ -0,0 +1,372 @@
+                    --danger-base: 339 97% 31%;
+                    --info-base: 217 92% 52%;
+                }
+
+                .menu_overlay {
+                    display: none;
+                    position: fixed;
+                    background: rgba(0,0,0,0.4);
+                    top: 0%;
+                    left: 0%;
+                    width: 100%;
+                    height: 100%;
+                    z-index: 500;
+                }
+	
+                .menuOpened.menu_overlay {
+                    display: block;
+                }
+
+                .cornerMenu {
+                    width: 50px;
+                    height: 50px;
+                    position: fixed;
+                    right: 25px;
+                    bottom: 25px;
+                    background: hsl(var(--background-step-1));
+                    box-shadow: 0px 3px 7px 0px rgba(0, 0, 0, 0.3);
+                    border-radius: 50%;
+                    border: 1px solid hsl(var(--border));
+                    text-align: center;
+                    line-height: 54px;
+                    color: #fff;
+                    font-size: 22px;
+                    z-index: 600;
+                    cursor: pointer;
+                }
+
+                .cornerMenu .avatar {
+                    z-index: 700;
+                }
+
+                .menu {
+                    /* Any lightness value below the threshold will result in white, any above will result in black */
+                    --switch: calc((var(--light) - var(--threshold)) * -100%);
+                    color: hsl(0, 0%, var(--switch));
+                    width: 32px;
+                    height: 32px;
+                    position: absolute;
+                    background-color: hsl(var(--background-step-2));
+                    box-shadow: 0px 3px 7px 0px rgba(0, 0, 0, 0.10);
+                    border-radius: 50%;
+                    text-align: center;
+                    line-height: 42px;
+                    font-size: 16px;
+                    top: -5px;
+                    right: 0;
+                    left: -5px;
+                    margin: auto;
+                    transform: translate(0, 0);
+                    opacity: 0;
+                    z-index: 550;
+                    transition: all .4s ease-in-out;
+                    padding: 8px;
+                    border: 1px solid hsl(var(--border));
+                    user-select: none;
+                    pointer-events: none;
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+
+                    svg {
+                        width: 24px;
+                        height: 24px;
+                    }
+                }
+
+                .menuOpened .menu {
+                    right: 0;
+                    opacity: 1;
+                    transition: translate .4s ease-in-out, background-color .15s ease;
```

Comment:
This one's on me, put the wrong keyword so animations when opening the menu broke
```suggestion
                    transition: transform .4s ease-in-out, background-color .15s ease;
```